### ISSUE TITLE
Any forks that are maintained?

### ISSUE THREAD

**Issue by sprite-1** [vysheng/tg#1570]
This hasn't had any pushed commits since 2016 so I think it's safe to assume it's been abandoned. I'm just wondering if someone forked this and is keeping it more up to date?

**Comment by erayerdin** [comment]
Not a fork, but [this](https://github.com/erayerdin/tgcli) is an alternative that I've recently created. It doesn't provide much functionality for now, just sending a message to a chat. Can provide further functionality though.

Also it is better tested.

**Comment by sprite-1** [comment]
Interesting project! I've starred it to keep an eye on it 👍

**Comment by andrew-phi** [comment]
Yeah, I would love to see this project revived too. So many pull requests - this is amazing.
I also fixed some things on my own, but the author seems nowhere to be found. :(

**Comment by berryamin** [comment]
Hello,
  I've just sent a Telegram message to vysheng, to apply as a maintainer for the project. I hope I will get some answer from him.

**Comment by berryamin** [comment]
Here is the active fork:
https://github.com/kenorb-contrib/tg

**Comment by AllanDaemon** [comment]
> Here is the active fork: https://github.com/kenorb-contrib/tg

It seems to not be active anymore.
